src/app/login/page.tsx

```
@@ -0,0 +1,236 @@
+                type="submit"
+                disabled={isLoading}
+                className="w-full flex justify-center py-3 px-4 border border-transparent rounded-xl shadow-sm text-sm font-semibold text-white bg-blue-600 hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 transition-all duration-200 transform hover:scale-[1.02] disabled:opacity-50 disabled:cursor-not-allowed"
+              >
+                {isLoading ? "Signing in..." : "Sign in to your account"}
+              </button>
+            </form>
+          </div>
+
+          <div className="px-8 py-4 bg-gray-50 border-t border-gray-100 text-center">
+            <p className="text-sm text-gray-600">
+              Don't have an account?{" "}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)